src/plugins/inspector/public/views/data/components/data_table.tsx

```diff
@@ -37,6 +37,10 @@ import { DataDownloadOptions } from './download_options';
 import { DataViewRow, DataViewColumn } from '../types';
 import { TabularData } from '../../../../common/adapters/data/types';
 import { IUiSettingsClient } from '../../../../../../core/public';
+import {
+  CSV_SEPARATOR_SETTINGS,
+  CSV_QUOTE_VALUES_SETTINGS,
+} from '../../../../../share/common/constants';
```

Comment:
Same note re: imports. We should only import from `share/public` here

Reply:
done